cmd/account/csp-enablement-account/csp-enablement-account.go

```diff
@@ -57,7 +57,7 @@ func newGet() *cobra.Command {
 
 	var getReq settings.GetCspEnablementAccountSettingRequest
 
-	cmd.Flags().StringVar(&getReq.Etag, "etag", getReq.Etag, `etag used for versioning.`)
+	cmd.Flags().StringVar(&getReq.Etag, "etag", getReq.Etag, `etag used for versioning. Wire name: 'etag'.`)
```

Comment:
We have this new "Wire name: '...'" string now everywhere, doesn't look intended / needed for CLI